ui/src/app/applications/components/application-sync-options/application-sync-options.tsx

```diff
@@ -5,7 +5,7 @@ import * as ReactForm from 'react-form';
 
 import './application-sync-options.scss';
 
-export const REPLACE_WARNING = `The resources will be synced using 'kubectl replace/create' command that is a potentially destructive action and might cause resources recreation.`;
+export const REPLACE_WARNING = `The resources will be synced using 'kubectl replace/create' command that is a potentially destructive action and might cause resources recreation. For example, it may cause a number of pods to be reset to min replicas resulting in system overload.`;
```

Comment:
What do you think about phrasing it in more natural-language terms, since it is a warning message to the user, something like:

....it might cause a number of pods to be reset to the minimum number of replicas.....

Reply:
Sounds good, I've updated the wording.